format-common/src/main/java/io/cdap/plugin/format/charset/fixedlength/FixedLengthCharsetTransformingDecompressorStream.java

```diff
@@ -104,4 +119,17 @@ protected int getCompressedData() throws IOException {
 
     return numReadBytes;
   }
+
+  /**
+   * Method that invokes the parent method to get decompressed data, without partition boundary awareness.
+   * <p>
+   * This method is visible for the purposes of testing the partition boundaries.
+   *
+   * @return number of bytes read from source input stream
+   * @throws IOException If there was a problem reading from the underlying input stream.
+   */
+  @VisibleForTesting
+  protected int getCompressedDataSuper() throws IOException {
```

Comment:
I see you committed this for test support, but I can't seem to find the test that uses it.

Reply:
I've added the missing class. I forgot to stage it for commit.